From db9f9e603fe5f9aa03bda9960aabb21e3cfdcc52 Mon Sep 17 00:00:00 2001
From: Jiajia Liu <liujiajia@kylinos.cn>
Date: Wed, 22 Jul 2026 15:18:19 +0800
Subject: [PATCH 06466/15333] wifi: cfg80211: reg: add a newline in DFS debug
 message

Add a missing newline in the debug message in print_regdomain.

Signed-off-by: Jiajia Liu <liujiajia@kylinos.cn>
Link: https://patch.msgid.link/20260722071819.22465-1-liujiajia@kylinos.cn
[break long line]
Signed-off-by: Johannes Berg <johannes.berg@intel.com>
---
 net/wireless/reg.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/net/wireless/reg.c b/net/wireless/reg.c
index 1e8214d6b6d8..a8336baf85dc 100644
--- a/net/wireless/reg.c
+++ b/net/wireless/reg.c
@@ -3792,7 +3792,8 @@ static void print_regdomain(const struct ieee80211_regdomain *rd)
 		}
 	}
 
-	pr_debug(" DFS Master region: %s", reg_dfs_region_str(rd->dfs_region));
+	pr_debug(" DFS Master region: %s\n",
+		 reg_dfs_region_str(rd->dfs_region));
 	print_rd_rules(rd);
 }
 
-- 
2.55.0

